librz/type/helpers.c

```diff
@@ -340,6 +340,28 @@ RZ_API bool rz_type_is_identifier(RZ_NONNULL const RzType *type) {
 	return type->kind == RZ_TYPE_KIND_IDENTIFIER;
 }
 
+/**
+ * \brief Checks if the RzType is strictly atomic
```

Comment:
What's the difference between an "atomic" type and a "strictly atomic" one??

Reply:
Atomic type (next function) is `char`, `char*`, `char****`, `char[20]`, `char[40][5][10]`, `char**[5][6]`, etc
Strictly atomic type is just `char`. Not a pointer or array of the atomic type.